src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/ComCaseloadService.kt

```diff
@@ -31,18 +34,18 @@ class ComCaseloadService(
     private const val PROBATION_SEARCH_BATCH_SIZE = 500
   }
 
-  fun getStaffCreateCaseload(deliusStaffIdentifier: Long): List<ManagedCase> {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Now that these are changed to the `ComCase` model, I presume these are used elsewhere otherwise we could remove the ManagedCase representation of the data?

Reply:
unfortunately we still need ManagedCase as an intermediate type as we collect date from delius and nomis and combine before we can transform to ComCase later. It's also used by CaCaseloadService.